paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -153,7 +154,7 @@ class CustomerSheet internal constructor(
                 onSuccess = {
                     CustomerSheetResult.Selected(it)
                 },
-                onFailure = { cause, _ ->
+                onFailure = { cause ->
                     CustomerSheetResult.Failed(cause)
                 }
             )
```

Comment:
Changes here are covered by the pre-existing `CustomerSheet` tests we added recently.